composites/sync-vip-prod/README.md

```diff
@@ -0,0 +1,147 @@
+```
+
+## Inputs
+
+| Input | Description | Required | Default |
+|-------|-------------|----------|---------|
+| `destination_repo` | Destination repository slug, including organization (e.g., `wpcomvip/pmc`) | Yes | - |
+| `destination_branch` | Branch name in the destination repository | Yes | - |
+| `destination_directory` | Directory path relative to repository root where files will be synced | Yes | - |
+| `protected_directories` | Directories to preserve during sync (typically brand-specific plugins) | No | `''` |
+| `GIT_EMAIL` | Email address for the Git user making commits | Yes | - |
+| `SLACK_WEBHOOK_URL` | Webhook URL for failure notifications | No | `''` |
+| `SSH_KEY` | SSH private key with access to both source and destination repositories | Yes | - |
+
+## Workflow Steps
+
+### 1. Repository Checkout
+- **Source Repository**: Checks out the current repository with minimal history (`fetch-depth: 1`)
+- **Destination Repository**: Checks out the specified VIP repository and branch using the provided SSH key
+- **Utility Scripts**: Checks out `penske-media-corp/pmc-jenkins-scripts` for helper functions
+
+### 2. File Synchronization Process
+
+The action uses several utility functions from the Jenkins scripts:
+
+- **`exclude_plugins`**: Temporarily moves protected directories out of the way
+- **`rsync_repo_files`**: Synchronizes files from source to destination using rsync
+- **`restore_plugins`**: Restores the protected directories after sync
+- **`setup_git_committer`**: Configures Git committer information based on the source repository
+
+### 3. Git Operations
+
+The action performs the following Git operations:
+- Configures Git user as `pmcvipgo-sync` with the provided email
+- Stages all changes (`git add --all`)
+- Checks if there are changes to commit
+- Creates a commit with metadata about the source repository and revision
+- Pushes changes to the destination branch
+
+### 4. Error Handling and Notifications
+
+If the sync fails and a Slack webhook URL is provided:
+- Retrieves job details from the GitHub API
+- Sends a formatted Slack notification with source, destination, and failure details
+
+## Commit Message Format
+
+Commits created by this action follow this format:
+```
+Sync from {source-repo}:{branch-name} for revision {commit-hash} by {author}
+```
+
+## Slack Notification
+
+When a sync fails, the action sends a Slack notification with:
+- Alert header indicating production sync failure
+- Source repository and branch information
+- Destination repository and branch information
+- Link to the failed GitHub Actions job
+
+## Protected Directories
+
+The `protected_directories` input allows you to specify directories that should be preserved in the destination repository during the sync. This is useful for:
+
+- Brand-specific plugins that aren't in the shared source repository
+- Configuration files specific to the VIP environment
+- Any custom modifications that shouldn't be overwritten
```

Comment:
These aren't the intended use cases. When it was in use, it was only for BGR and Gold Derby as they had plugins added directly to their VIP repos. We rectified that for GD during the replatform, while Static Media resolved the BGR situation.

```suggestion
```